src/legacy-replacements/Button/Button.tsx

```diff
@@ -44,6 +44,15 @@ export function ButtonReplacement ({
         }
         debugMsg.forEach(element => console.debug(element));
     }
+    if (typeof otherProps.accent !== "undefined") {
```

Comment:
Can we remove the if expressions and just keep the delete lines?

Reply:
No, if I remember correctly Typescript throws errors when undefined properties were deleted.